openapi/single-sign-on/single-sign-on.yaml

```diff
@@ -0,0 +1,384 @@
+            type: string
+          in: query
+          name: code_verifier
+          description: 'When using `authorization_code`: The PKCE code verifier for the `code_challenge` presented to the authorization endpoint starting this flow. This is used to protect against security attacks and should be used by all, but [is required for public clients].'
+      tags:
+        - Single-Sign-On
+      security: []
+  /oauth2/user-info:
+    get:
+      summary: OpenID User Info
+      operationId: openid-userinfo
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: ''
+                type: object
+                x-examples:
+                  example-1:
+                    sub: '248289761001'
+                    name: Jane Doe
+                    given_name: Jane
+                    family_name: Doe
+                    preferred_username: j.doe
+                    email: janedoe@example.com
+                    picture: 'http://example.com/janedoe/me.jpg'
+                properties:
+                  sub:
+                    type: string
+                    minLength: 1
+                  name:
+                    type: string
+                    minLength: 1
+                  given_name:
+                    type: string
+                    minLength: 1
+                  family_name:
+                    type: string
+                    minLength: 1
+                  preferred_username:
+                    type: string
+                    minLength: 1
+                  email:
+                    type: string
+                    minLength: 1
+                  picture:
+                    type: string
+                    minLength: 1
+                  namespace:
+                    type: string
+                  product_navbar_data_url:
+                    type: string
+                required:
+                  - sub
+                  - name
+                  - given_name
+                  - family_name
+                  - preferred_username
+                  - email
+                  - picture
+          headers: {}
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        This is the OpenID UserInfo endpoint, used for obtaining scoped identity data relating to the bearer token's associated User.
+
+        Requires an access token scoped to at least one of the following supported scopes: `openid`, `email`, `profile`.
+
+        `openid` or any other supported scope returns the basic user info response:
+        - the `sub` field (this is the User's id)
+        - the `namespace` which that User is stored within
+        - the `product_navbar_data_url` which is a URL which can be used to embed Vendasta's Product Navbar in your application [reference needed].
```

Comment:
`product_navbar_data_url` gets documented in description, it seems like the wrong spot